config/jobs/image-pushing/k8s-staging-kwok.yaml

```diff
@@ -0,0 +1,26 @@
+postsubmits:
+  # this is the github repo we'll build from; this block needs to be repeated for each repo.
+  kubernetes-sigs/kwok:
+    - name: post-kwok-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # this is the name of some testgrid dashboard to report to.
+        testgrid-dashboards: sig-scheduling, sig-k8s-infra-gcb
+      decorate: true
+      branches:
+        # Build tags also
```

Comment:
I noticed other sig-scheduling owned configs ([1](https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-scheduler-plugins.yaml), [2](https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-kueue.yaml)) carry the following lines:

```
        - ^master$
        - ^release-.*
```

Reply:
No release and master, I was just trying to be lazy, now I'm making up for it
https://github.com/kubernetes-sigs/kwok/pull/18/commits/f5b23506db3c15055abbec9f22cfa99756d0c9ab